aids.niaiddata.org/etlMapping.yaml

```diff
@@ -25,14 +25,43 @@ mappings:
       - path: hiv_history_records
         props:
           - name: arthxbase
+          - name: bsdate
           - name: cd4nadir
+          - name: ehaart
+          - name: fcd4lowd
           - name: fposdate
+          - name: frstaidd
+          - name: frstartd
           - name: frstdthd
+          - name: frsthaad
+          - name: lastafrd
+          - name: lastcond
+          - name: lastcontact
+          - name: lastnoad
+          - name: lastnohd
+          - name: lcd4higd
+          - name: lnegdate
           - name: status
       - path: comorbidities
         props:
           - name: bshbvstat
           - name: bshcvstat
+          - name: frstcncrd
+          - name: frstdmd
+          - name: frstdmmd
+          - name: frsthtnd
+          - name: frsthtnmd
+      - path: diagnoses
```

Comment:
ETL is not happy with this part:
`the properties '[{'name': 'primary_diagnosis'}, {'name': 'comorbidity'}, {'name': 'comorbidity_anemia'}, {'name': 'comorbidity_diabetes'}, {'name': 'comorbidity_hepatic_disease'}, {'name': 'comorbidity_hiv'}, {'name': 'comorbidity_renal_disease'}, {'name': 'prior_treatment'}, {'name': 'year_of_diagnosis'}]' for 'diagnoses'
for parent 'diagnosis'
has multiplicity 'many_to_one' that cannot be used on in 'flatten_props'`

so we need to put everthing here into `aggregated_props`

Reply:
you can sort it to grab the most recent one

Reply:
like this for example: https://github.com/uc-cdis/cdis-manifest/blob/master/gen3.datastage.io/etlMapping.yaml#L35